src/libs/API.js

```diff
@@ -4,6 +4,20 @@ import * as Request from './Request';
 import * as SequentialQueue from './Network/SequentialQueue';
 import {version} from '../../package.json';
 
+/**
+ * Function for WRITE commands.
```

Comment:
```suggestion
/**
```
Not really necessary and doesn't provide any value IMO